themes/bootstrap5/templates/_ui/components/page-title.phtml

```diff
@@ -0,0 +1,15 @@
+<?php
+  $title = $this->title ?? 'title_missing';
+  $headerAttrs = [
+    'class' => $this->_componentClass,
+  ];
+  if ($this->inLightbox) {
+    $headerAttrs['tabindex'] = '0';
+    $headerAttrs['class'] .= ' lightbox-title';
+  }
+  $level = $this->level ?: 2;
```

Comment:
Would it make sense to validate that `$this->level` is an integer? Maybe that's unnecessary work, but it would prevent somebody from doing something weird with a string value. You could either wrap `intval` around this (so weird truthy values would turn into '1') or you could add a check (perhaps an `assert` call to keep it low-cost) to validate. Or you could do nothing if you don't think it's worth the effort. What do you think?

Reply:
I think it could be better